### ISSUE TITLE
Repost: Ghosts can control slimes.

### ISSUE THREAD

**Issue by ghost** [yogstation13/yogstation-classic#57]
Step 1: Clickdrag slime as a ghost.

Step 2: Clickdrag it onto the xenobiologist.

Step 3: It latches onto the xenobiologist.

Step 4: Be unlogged and a great way for ghosts to grief.

**Comment by Hydre** [comment]
How is this even possible

**Comment by X-TheDark** [comment]
This is because the "/mob/living/simple_animal/slime/MouseDrop" proc in [slime.dm](https://github.com/yogstation13/yogstation/blob/c8b5bf85b54c7c0adb059925829a6246fd115999/code/modules/mob/living/simple_animal/slime/slime.dm) (line 180) doesn't check for who drags the slime over.

Solution, change the code from this:

```
if(isliving(A) && A != src)
```

to this (tg fixed this as well):

```
if(isliving(A) && A != src && usr == src)
```

Heck, if you want to be fancy, you could add a check to see if the one who drags the slime is the slimes friend (obviously excluding ghosts from the selection), so you can make the slime feed on someone if you are it's friend, but that would be a different issue (feature request) rather then the bug.

**Comment by ghost** [comment]
This might want to be fixed fast, two days ago some dude used it to kill a traitor xenobio who spaced him, couldn't catch the name of the ghost who did it but the xenobio ahelped about being attacked by a setinent slime which was screaming 'I'M NOT DOING THIS' and running away but kept getting dragged on.

**Comment by TehFlaminTaco** [comment]
Resolved.

**Comment by george99g** [comment]
Fixed by @TehFlaminTaco with #68
